UPGRADE-API-1.13.md

```diff
@@ -63,166 +129,122 @@
-   * `Sylius\Bundle\ApiBundle\Filter\Doctrine\TaxonFilter`
-   * `Sylius\Bundle\ApiBundle\Serializer\ChannelPriceHistoryConfigDenormalizer`
-   * `Sylius\Bundle\ApiBundle\Serializer\ProductNormalizer`
-   * `Sylius\Bundle\ApiBundle\Serializer\ProductVariantNormalizer`
-   * `Sylius\Bundle\ApiBundle\Serializer\ZoneDenormalizer`
-
-1. The `Sylius\Bundle\ApiBundle\Filter\Doctrine\CatalogPromotionChannelFilter` service and class has been renamed to `Sylius\Bundle\ApiBundle\Filter\Doctrine\ChannelsAwareChannelFilter`.
-
-1. The `sylius.api.product_taxon_filter` filter has been removed and its functionality has been superseded by the `sylius.api.search_filter.taxon.code` filter. The usage stays the same.
+    * `Sylius\Bundle\ApiBundle\OpenApi\Documentation\AdminAuthenticationTokenDocumentationModifier`
+    * `Sylius\Bundle\ApiBundle\OpenApi\Documentation\ShopAuthenticationTokenDocumentationModifier`
+    * `Sylius\Bundle\ApiBundle\OpenApi\Documentation\ProductDocumentationModifier`
+    * `Sylius\Bundle\ApiBundle\OpenApi\Documentation\ProductImageDocumentationModifier`
+    * `Sylius\Bundle\ApiBundle\OpenApi\Documentation\ProductSlugDocumentationModifier`
+    * `Sylius\Bundle\ApiBundle\OpenApi\Documentation\ProductVariantDocumentationModifier`
+    * `Sylius\Bundle\ApiBundle\OpenApi\Documentation\ShippingMethodDocumentationModifier`
+    * `Sylius\Bundle\ApiBundle\OpenApi\Documentation\PathHiderDocumentationModifier`
+    * `Sylius\Bundle\ApiBundle\OpenApi\Documentation\AcceptLanguageHeaderDocumentationModifier`
+
+1. All usages of `ApiPlatform\Core\Api\IriConverterInterface` have been switched to its non-deprecated
+   counterpart `ApiPlatform\Api\IriConverterInterface`.
+   Due to that, the constructor and usage in the following classes have been changed accordingly:
+    * `Sylius\Bundle\ApiBundle\Controller\GetProductBySlugAction`
+    * `Sylius\Bundle\ApiBundle\Controller\UploadAvatarImageAction`
+    * `Sylius\Bundle\ApiBundle\EventListener\AdminAuthenticationSuccessListener`
+    * `Sylius\Bundle\ApiBundle\EventListener\AuthenticationSuccessListener`
+    * `Sylius\Bundle\ApiBundle\Filter\Doctrine\CatalogPromotionChannelFilter`
+    * `Sylius\Bundle\ApiBundle\Filter\Doctrine\ProductVariantCatalogPromotionFilter`
+    * `Sylius\Bundle\ApiBundle\Filter\Doctrine\ProductVariantOptionValueFilter`
+    * `Sylius\Bundle\ApiBundle\Filter\Doctrine\ProductVariantOptionValueFilter`
+    * `Sylius\Bundle\ApiBundle\Filter\Doctrine\TaxonFilter`
+    * `Sylius\Bundle\ApiBundle\Serializer\ChannelPriceHistoryConfigDenormalizer`
+    * `Sylius\Bundle\ApiBundle\Serializer\ProductNormalizer`
+    * `Sylius\Bundle\ApiBundle\Serializer\ProductVariantNormalizer`
+    * `Sylius\Bundle\ApiBundle\Serializer\ZoneDenormalizer`
+
+1. The `Sylius\Bundle\ApiBundle\Filter\Doctrine\CatalogPromotionChannelFilter` service and class has been renamed
+   to `Sylius\Bundle\ApiBundle\Filter\Doctrine\ChannelsAwareChannelFilter`.
+
+### Endpoints changes
```

Comment:
```suggestion
### Resources changes
```
or
```suggestion
### Usage changes
```
or something like that.
There are more than just endpoints' differences described in this section.